src/pages/filter/common/dashboard/components/body/dashboard-body.tsx

```diff
@@ -162,42 +47,21 @@ export const DashboardBody = observer(
           draggableHandle="[data-drag-handle]"
           isResizable={false}
           className={styles.body__gridLayout}
-          onDragStop={handleLayoutChange}
+          onLayoutChange={layoutChange}
+          onResize={changeTabsHeight}
```

Comment:
Are you sure it's needed? We have isResizable={false} and changeTabsHeight() is triggered in the 35th string every time we change tab height

Reply:
removed